tests/yam/validate/validate_registered_products.pm

```diff
@@ -0,0 +1,36 @@
+# SUSE's openQA tests
```

Comment:
the idea was to adapt validate_registered_addons and rename it to this name you used here.
Also don't forget to avoid creating new settings like 'PRODUCTS'.

Reply:
The name changed to product caused confuse, because 'sles' is a product, while ha is extension, so in fact the module is verify addons and sles registered after installation.  If must in the same module, could it be 'validate_registered_product_and_addons.pm'?